### PR TITLE
Some typos and supplementation in sentence structures

### PR DESCRIPTION
**Pull request by ErikXXon**
Correct some typos and supplementation in sentence structures (Swedish).

### CODE REVIEW

File: config/locales/sv.yml [hunk 1]
```diff
@@ -117,6 +117,13 @@ sv:
       unsubscribe: Säga upp
       username: Användarnamn
       web: Webb
+    account_moderation_notes:
```
Comment (ykzts) [inline config/locales/sv.yml]: duplicate of [line 53](https://github.com/tootsuite/mastodon/pull/5441/files#diff-3c1154d73d9ad84a41fa6fcc6a890981L53)

File: config/locales/sv.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 sv:
   about:
-    about_hashtag_html: Dessa är offentliga toots märkta med <strong>#%{hashtag}</strong>. Du kan interagera med dem om du har ett konto någonstans i federationen.
     about_mastodon_html: Mastodon är ett socialt nätverk baserat på öppna webbprotokoll och gratis, öppen källkodsprogramvara. Det är decentraliserat som e-post
+    about_hashtag_html: Dessa är offentliga toots märkta med <strong>#%{hashtag}</strong>. Du kan interagera med dem om du har ett konto någonstans i federationen.
```
Comment (ykzts) [inline config/locales/sv.yml]: Why did you move this message?

File: config/locales/simple_form.sv.yml [hunk 3]
```diff
@@ -59,9 +59,9 @@ sv:
...
         mention: Skicka e-post när någon nämner dig
-        reblog: Skicka e-post när någon boostar din status
-    'no': Nej
+        reblog: Skicka e-post när någon knuffar din status
+    'no': 'Nej'
     required:
       mark: "*"
       text: obligatorisk
-    'yes': Ja
+    'yes': 'Ja'
```
Comment (ykzts) [inline config/locales/simple_form.sv.yml]: `'` is unnecessary.

File: config/locales/simple_form.sv.yml [hunk 4]
```diff
@@ -59,9 +59,9 @@ sv:
         follow: Skicka e-post när någon följer dig
         follow_request: Skicka e-post när någon begär att följa dig
         mention: Skicka e-post när någon nämner dig
-        reblog: Skicka e-post när någon boostar din status
-    'no': Nej
+        reblog: Skicka e-post när någon knuffar din status
+    'no': 'Nej'
```
Comment (ykzts) [inline config/locales/simple_form.sv.yml]: `'` is unnecessary.

**Review by ykzts — DISMISSED** [review]

File: config/locales/sv.yml (hunk 2 above)
Comment (ErikXXon) [reply config/locales/sv.yml]: Freaky fingers allowed me to add the line again, maybe wrong line then.

Comment (ykzts) [reply config/locales/sv.yml]: The messages of the locale file must be arranged in alphabetical order. Can you return to the original position?

**Review by Gargron — APPROVED** [review]
